src/omotes_orchestrator/main.py

```diff
@@ -13,13 +13,15 @@
 from omotes_sdk.internal.orchestrator_worker_events.messages.task_pb2 import (
     TaskResult,
     TaskProgressUpdate,
+    TaskEsdlMessage,
```

Comment:
I wondered if it is possible and reasonable to define one common EsdlMessage protocol in SDK and reuse it in different places. As TaskEsdlMessage and EsdlMessage have the same definition. Maybe with import? https://protobuf.dev/programming-guides/proto3/#importing